apps/meteor/app/livechat/server/lib/hooks.ts

```diff
@@ -109,3 +110,11 @@ export const onNewRoom = makeFunction(async (room: IOmnichannelRoom) => {
 		await sendToCRM('LivechatSessionStart', room);
 	}
 });
+
+export const afterTakeInquiry = makeFunction(
+	async (params: { inquiry: InquiryWithAgentInfo; room: IOmnichannelRoom }, _agent: { agentId: string; username: string }) => {
```

Comment:
since this is not a callback anymore, I'd suggest we make it a single param with three fields, it doesn't make much sense for a function to a have a second param if the first one is already a "collection" (like an object with very distinct fields)
```suggestion
	async (params: { inquiry: InquiryWithAgentInfo; room: IOmnichannelRoom, agent?: { agentId: string; username: string }) => {
```